### PR TITLE
[Mark|Mark-Scan] Replace text on IdlePage screens with <UiString>s

### PR DESCRIPTION
**Pull request by kofi-q**
## Overview

Converting text on the BMD `IdlePage` screens to use the language-aware `<UiString>`. 

Some wording changes were needed to enable better-sounding audio, but might need some tweaking in the future.

## Demo Video or Screenshot
| Default  | Sample Translation |
| ------------- | ------------- |
| Screenshot 2023-11-07 at 12 32 17 | Screenshot 2023-11-07 at 12 33 39 |
| Screenshot 2023-11-07 at 12 32 41 | Screenshot 2023-11-07 at 12 34 05 |
| Screenshot 2023-11-07 at 12 34 28 | Screenshot 2023-11-07 at 12 35 53 |

## Testing Plan
- Manual

## Checklist

- ~[ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.~

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/mark-flow-ui/src/pages/idle_page.tsx [hunk 1]
```diff
@@ -31,40 +41,54 @@ export function IdlePage({
...
-            <Button variant="primary" onPress={onDismiss}>
-              Yes, I’m still voting.
+          <Font align="center">
+            {/*
+             * TODO(kofi): the old "audiofocus" pattern only works for one-time
+             * readouts of static text. Need to introduce configurable
+             * functionality for re-initiating speech when text gets
+             * added/replaced (e.g. when we move from this text to
+             * "Clearing ballot" above).
+             */}
```
Comment (arsalansufi) [inline libs/mark-flow-ui/src/pages/idle_page.tsx:70]: Good to know!

File: apps/mark-scan/frontend/src/components/ballot.tsx [hunk 2]
```diff
@@ -12,6 +16,16 @@ import { ReviewScreen } from '../pages/review_screen';
 import { StartScreen } from '../pages/start_screen';
 import { ContinueToReviewPage } from '../pages/continue_to_review_page';
 
+const USER_ACTIVITY_EVENT_TYPES: EventsType[] = (() => {
+  const allEvents = new Set(DEFAULT_EVENTS);
+  // The IdlePage has an autofocus button to enable gamepad interaction,
+  // ends up triggering a focus event that would reset the idle timer.
```
Comment (arsalansufi) [inline apps/mark-scan/frontend/src/components/ballot.tsx:22]: I think this comment is missing a word, maybe "that" or "which":

```
// The IdlePage has an autofocus button to enable gamepad interaction,
// which ends up triggering a focus event that would reset the idle timer.
```

**Review by arsalansufi — APPROVED** [review]
LGTM!

File: apps/mark-scan/frontend/src/components/ballot.tsx (hunk 2 above)
Comment (kofi-q) [reply apps/mark-scan/frontend/src/components/ballot.tsx]: Thanks!

**Pull request merged**